xml/common_intro_feedback_i.xml

```diff
@@ -16,66 +16,69 @@
  </info>
 
  <para>
-  Several feedback channels are available:
+  Your feedback and contribution to this documentation is welcome!
+  Several channels are available:
  </para>
 
  <variablelist>
   <varlistentry os="sles;sled">
-   <term>Bugs and Enhancement Requests</term>
+   <term>Service Requests and Support</term>
    <listitem>
-    <para os="sles;sled">
+    <para>
      For services and support options available for your product, refer to
      <link xlink:href="http://www.suse.com/support/"/>.
     </para>
     <para>
-     Help for &opensuse; is provided by the community. Refer to <link
-     xlink:href="https://en.opensuse.org/Portal:Support"/> for more information.
-    </para>
-    <para os="sles;sled">
-<!--taroth 2015-06-01: http://bugzilla.suse.com/show_bug.cgi?id=882059-->
-     To report bugs for a product component, go to
-     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and
-     click <guimenu>Create New</guimenu>.
-    </para>
-    <para os="osuse">
-     To report bugs &productname;, go to <link
-     xlink:href="https://bugzilla.opensuse.org/"/>, log in, and click
-     <guimenu>New</guimenu>.
+     To open a service request, you need a subscription at &scc;. Go to
+     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and click
+     <guimenu>Create New</guimenu>.
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="osuse">
+  <varlistentry>
    <term>Bug Reports</term>
    <listitem>
     <para>
-     To report bugs for &productname;, go to
-     <link xlink:href="https://bugzilla.opensuse.org/"/>, log in, and
-     click <guimenu>New</guimenu>.
+      To report documentation bugs in <phrase os="sles;sled">
+      <link xlink:href="https://bugzilla.suse.com/"/>
+     </phrase><phrase os="osuse">
+      <link xlink:href="https://bugzilla.opensuse.org/"/>
+     </phrase>, use the links next to each headline in the HTML version of this
```

Comment:
The `<phrase>` here is probably redundant. You can directly add the `os="sles;sled"` attribute on the `<link/>` element. To avoid any whitespace issues, concatenate all `<link/>` elements without spaces.

As such, I would propose:

```xml
To report documentation bugs in <link os="sles;sled" xlink:href="https://bugzilla.suse.com/"
/><link os="osuse" xlink:href="https://bugzilla.opensuse.org/"/>, use the links next to each
headline in the HTML version of this
```

Unfortunately, the profiling attributes make the above paragraph not very readable. So I guess, it's a matter of taste if you prefer with or without `<phrase>`s.

Reply:
Thanks, I have added the profiling attributes directly to the ```<link/>``` element now.